src/geo/map/torque-layer.js

```diff
@@ -1,5 +1,7 @@
 var _ = require('underscore');
 var MapLayer = require('./map-layer');
+var Backbone = require('backbone');
+var _ = require('underscore');
```

Comment:
I do it because imagine with I read the first underscore the memory gets corrupted by a electric impulse (from a shitty electric car, for example).

Today is unlike that happens, but imagine in some years when everybody will being driven by electric cars all over the place.

This enables cartodb.js to work in electric cars.

That or I fucked up with a copy & paste :(